include/oneapi/dpl/pstl/iterator_impl.h

```diff
@@ -695,6 +703,18 @@ struct ignore_copyable
     {
         return *this;
     }
+
+    bool
+    operator==(const ignore_copyable& other) const
+    {
+        return true;
+    }
+
+    bool
+    operator!=(const ignore_copyable& other) const
```

Comment:
Does implementation through operator== according to common practice make sense?

Reply:
It's good to be consistent with the practice even with such a trivial implementation.

Reply:
done.